test/itkFrequencyBandPassImageFilterTest.cxx

```diff
@@ -72,12 +72,10 @@ int itkFrequencyBandPassImageFilterTest( int, char*[] )
   TEST_SET_GET_VALUE( passBand, bandPassFilter->GetPassHighFrequencyThreshold() );
 
   bool passLowFreqThreshold = true;
-  bandPassFilter->SetPassLowFrequencyThreshold( passLowFreqThreshold );
-  TEST_SET_GET_VALUE( passLowFreqThreshold, bandPassFilter->GetPassLowFrequencyThreshold() );
+  TEST_SET_GET_BOOLEAN( bandPassFilter, PassLowFrequencyThreshold, passLowFreqThreshold );
```

Comment:
@phcerdan I think it does (did), indeed:
https://github.com/phcerdan/ITKIsotropicWavelets/pull/12/files/9385754552b7bd8b51ac18352934ce461ceb2112#diff-e7e468398fe0619396f5bb9cd48ac60cR105

If all the Set/Get and BooleanMacros are defined for a boolean member, this should not be an issue. I compiled locally before pushing. Anyway, looks like it is not an issue anymore. Thanks for merging !